bw2data/project.py

```diff
@@ -201,6 +201,37 @@ def _load_revisions(
                 revs.append(json.load(f))
         return head, revs
 
+    def _rebase(
```

Comment:
```suggestion
    def _rebase_porcelain(
```
This just makes it less likely anyone changes this assuming it's a plumbing command.

Reply:
I can't think of a good way to do this, so I've just decided to leave an ominously large documentation string in there.